test/unittests/execute_floating_point_test.cpp

```diff
@@ -121,9 +92,21 @@ template <typename T>
 class execute_floating_point_types : public testing::Test
 {
 protected:
+    using L = typename FP<T>::Limits;
+
+    // The q is any positive floating-point value without zeros, infinities and NaNs.
```

Comment:
Why "q"?

Reply:
> The meta variable q is used to range over (signless) rational magnitudes, excluding nan or ∞.
https://webassembly.github.io/spec/core/exec/numerics.html#numerics

Reply:
That includes zeroes, but it's not so important

Reply:
I would call these two maybe `positive_special_values` and `ordered_special_values`

Reply:
Done.